Resources/Textures/Structures/Wallmounts/barsign.rsi/meta.json

```
@@ -7,6 +7,7 @@
   "license": "CC-BY-SA-3.0",
   "copyright": "https://github.com/vgstation-coders/vgstation13/commit/1fbb5e417d0200fe60c6b628656a371093d2d715",
   "copyright": "https://github.com/tgstation/tgstation/commit/68aade356cf53032f206cb5a887581ac7d166cae",
+  "copyright": "https://github.com/space-wizards/space-station-14/commit/b301d3ac26cb2301048eda4535667c28e2283d41",
```

```suggestion
  "copyright": "https://github.com/space-wizards/space-station-14/commit/b301d3ac26cb2301048eda4535667c28e2283d41 - Made by ProPandaBear for SS14",
```

You don't need to add your own commit for this but I guess that works. You should add something like:

Made by githubname for SS14

---

Awesome, thanks for the help. Went ahead and committed your suggestion.